### DISCUSSION TITLE
유지보수 게시물 조회 API 더미데이터 요청합니다

### DISCUSSION THREAD

**Discussion by yerim08** [10aeat/10aeat_Backend#165] [FE -> BE 요청사항]
각 케이스 별 더미데이터가 필요합니당 ,, 

더미데이터를 넣어주실 수 있으신지, 혹은 현재 어렵다면 언제까지 넣어주실 수 있는지 여쭤봅니다!

데이터가 없으니 필터링 기능을 제대로 만들지 못하고 있습니다

유지보수 게시물 조회 API 이외에도 현재 데이터가 거의 없기 때문에 제대로 된 개발에 한계가 있는 상태입니다 ㅠ 

실제 데이터를 받아서 사용했을 때 생길 수 있는 여러 상황을 대비해야 하기 때문에 더미데이터가 입력된 API는 공지를 해주시면 감사드리겠습니당 !

<우선 필요한 데이터 케이스 : 유지보수 게시물 조회 API>
카드 버전에 따른 데이터, 카테고리, 진행상황 등 고려

--> 진행상황과 카테고리 별로 
- 이미지, 기간 모두 있는 버전
- 이미지는 없고 기간은 있는 버전
- 이미지는 있고 기간은 없는 버전
- 이미지도 기간도 없는 버전

**Comment by ghrltjdtprbs** [comment]
7시 40분까지 더미데이터 넣겠습니다! 참고로 이미지 부분은 백엔드측에서 완성이 안된 상태이기 때문에 현재 이미지 url만 응답값으로 간다는 점 참고해주세요!

**Reply by yerim08** [reply]
혹시 8시 40분까지인가요 ???

**Reply by ghrltjdtprbs** [reply]
녱 최대한 더미 데이터 종류별로 넣어드리겠습니당 ㅎㅎ 그 외에 더미데이터가 필요하시다면 말씀해주세요!

**Reply by yerim08** [reply]
네 감사합니다 ㅎㅎ !!

**Reply by ghrltjdtprbs** [reply]
요청주신 부분 완료했습니다! 
더미데이터라서 category, progress 부분을 제외한 데이터는 동일합니다! 
참고로 유지보수는 redDot 기능을 백엔드에서 처리하기 때문에 테스트 해보시라고 게시글 이슈도 무작위로 몇 개 생성했습니다! 😊

**Reply by yerim08** [reply]
네! 감사합니다 😆
